core/src/main/java/com/novoda/espresso/AccessibilityRules.java

```diff
@@ -6,12 +6,12 @@
 import android.support.test.InstrumentationRegistry;
 import android.view.accessibility.AccessibilityManager;
 
+import com.novoda.accessibility.Service;
+
 import org.junit.rules.TestRule;
 
 import java.util.List;
 
-import static com.novoda.espresso.AccessibilityServiceToggler.Service;
```

Comment:
we're now using the one from [accessibilitools 1.5.0](github.com/novoda/accessibilitools)